File: Make array consecutive 2/array-consecutive-2.py
def makeArrayConsecutive2(statues):
    """
    :rtype statues: list
    :return: int
    """
    return len(range(min(statues), max(statues) + 1)) - len(statues)


print(makeArrayConsecutive2([6, 2, 3, 8]))  # 3
print(makeArrayConsecutive2([0, 3]))  # 2
print(makeArrayConsecutive2([5, 4, 6]))  # 0

File: Make array consecutive 2/array-consecutive-2.js
/**
 * Make array consecutive 2
 *
 * @return {number}
 * @param {array} statues
 */
function makeArrayConsecutive2(statues) {
  return Math.max(...statues) - Math.min(...statues) - statues.length + 1
}

console.log(makeArrayConsecutive2([6, 2, 3, 8])); // 3
console.log(makeArrayConsecutive2([0, 3])); // 2
console.log(makeArrayConsecutive2([5, 4, 6])); // 0